test/e2e/far_e2e_test.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 const (
-	// testNamespace  = "fence-agents-remediation"
-	testNamespace  = "openshift-operators"
+	testNamespace = "fence-agents-remediation"
+	// testNamespace  = "openshift-operators"
```

Comment:
Is there a reason to keep this commented code?

Reply:
Not really 😄